build.sbt

```diff
@@ -3,12 +3,13 @@ import sbtghactions.UseRef
 
 val Scala213 = "2.13.5"
 val Scala212 = "2.12.13"
+val Scala3   = "3.0.0"
 
 enablePlugins(SonatypeCiReleasePlugin)
 
 ThisBuild / organization := "org.typelevel"
 ThisBuild / baseVersion := "1.3"
-ThisBuild / crossScalaVersions := Seq(Scala213, Scala212, "3.0.0-RC2", "3.0.0-RC3")
+ThisBuild / crossScalaVersions := Seq(Scala3, Scala213, Scala212)
```

Comment:
Does reordering these make Scala 3 the default version?  If so, does that irritate tooling?

Reply:
Did a quick double check crossScalaVersions only influences the order in which `++` prefixed tasks set the version which to run. 

And the default Scala version when you open up sbt  is set by the `scalaVersion` property.

I guess it makes sense for the first one in x-compilation to be the same  as `scalaVersion`

Reply:
Oh, that's right.  I'm probably thinking of some org plugin I've used somewhere that infers it from the cross versions.